torch/functional.py

```diff
@@ -191,10 +195,12 @@ def split(
 
 
 def einsum(*args: Any) -> Tensor:
-    r"""einsum(equation, *operands) -> Tensor
```

Comment:
we can leave this as is and add a comment: # noqa D400 - which should skip this line.